src/app/demo/demo.module.ts

```diff
@@ -16,28 +16,28 @@ export const routes = [
     path: 'typescript',
     loadChildren: './typescript/typescript.module#TypescriptModule',
     name: 'Typescript',
-    description: 'Learn TypeScript: A language that extends JavaScript, which angular is built with.',
+    description: 'Learn TypeScript: A superset of JavaScript, which Angular is written in',
```

Comment:
Oh I see; I knew what superset meant before and I didn't know TS and don't have a math background. To me extends sounded more programmy and less specific/harder to understand 😅